cypress/e2e/seededFlows/articleFlows/postSidebarActions.spec.js

```diff
@@ -92,12 +92,9 @@ describe('Post sidebar actions', () => {
       const originalScroll = comments.scrollIntoView.bind(comments);
       cy.stub(comments, 'scrollIntoView').callsFake(() => originalScroll());
 
-      cy.findByRole('button', { name: 'Jump to Comments' })
-        .click()
-        .then(() => {
-          cy.get('@commentsSortDropdown').should('be.within_viewport');
-          cy.get('@articleHeader').should('not.be.within_viewport');
-        });
+      cy.findByRole('button', { name: 'Jump to Comments' }).click();
+      cy.get('@commentsSortDropdown').should('be.within_viewport');
+      cy.get('@articleHeader').should('not.be.within_viewport');
```

Comment:
Removing the `then` here may break the spec. Again, https://docs.cypress.io/guides/core-concepts/retry-ability#Actions-should-be-at-the-end-of-chains-not-the-middle has another way but I was mostly focusing on cleaning up all the red squiggly lines with the first pass here.